proposals/0000-reduce-fragmentation.md

```diff
@@ -0,0 +1,837 @@
+
+* [ ] `article` is equivalent to `View` with `display:block` layout and `role="article"`.
+* [ ] `aside`
+* [ ] `div` is equivalent to `View` with `display:block` layout.
+* [ ] `footer`.
+* [ ] `h1-6`.
+* [ ] `header`.
+* [ ] `main`.
+* [ ] `nav`.
+* [ ] `section` is equivalent to `View` with `display:block` layout and `role="section"`.
+
+## Text
+
+Text content elements organize blocks or sections of content. Important for accessibility, these elements identify the purpose or structure of that content.
+
+* [ ] `blockquote`.
+* [ ] `li`.
+* [ ] `ol`.
+* [ ] `ul`.
+* [ ] `p`.
+* [ ] `pre`.
+
+## Inline text
+
+Inline text semantics define the meaning, structure, or style of a word, line, or any arbitrary piece of text.
+
+* [ ] `a`.
+* [ ] `b`.
+* [ ] `bdi`.
+* [ ] `bdo`.
+* [ ] `code`.
+* [ ] `em`.
+* [ ] `span` is equivalent to `Text` with `display:inline` layout.
+* [ ] `strong`.
+* [ ] `sub`.
+* [ ] `sup`.
+
+## Media
+
+* [ ] `img` is equivalent to `Image` without advanced loader configuration.
+
+## Forms
+
+Elements which can be used together to create forms which the user can fill out and submit. There's a great deal of further information about this available in the [HTML forms guide](https://developer.mozilla.org/en-US/docs/Learn/Forms).
+
+* [ ] `button`.
+* [ ] `input` is equivalent to `TextInput` with `inputMode` corresponding to `type`.
+* [ ] `label`.
+* [ ] `optgroup`.
+* [ ] `option`.
+* [ ] `progress`.
+* [ ] `select`.
+* [ ] `textarea` is equivalent to `TextInput` with `multiline={true}`.
+
+-----
+
+# Elements API (DOM subset)
+
+Subset of cross-platform DOM APIs exposed on element instances, document, etc. This would be an imperative API for working with the Shadow tree in React Native. Based on data from [MDN Web API](https://developer.mozilla.org/en-US/docs/Web/API), please refer to the MDN links for complete DOM APIs and comment if you believe any should be included in the proposed subset.
+
+In practice, this DOM API subset is generally limited to the “best practice” APIs that can safely be used with React DOM (i.e., read-only operations that don’t modify the DOM tree), and most React components should already be limited to this subset.
+
+## EventTarget
+
+The `EventTarget` API is implemented in JS environments, and should be available on React Native host elements. This feature is commonly used on web, and could be used by React Native developers to support more complex features without first requiring further core API changes to support each use case.
+
+### Methods
+
+- [ ] `EventTarget.addEventListener()`. Registers an event handler to a specific event type on the element.
+- [ ] `EventTarget.dispatchEvent()`. Dispatches an event to this node in the DOM and returns a boolean value that indicates whether no handler canceled the event.
+- [ ] `EventTarget.removeEventListener()`. Removes an event listener from the element.
```

Comment:
We already have an `unstable_*` implementation of the listener methods in React Native. We'd need to promote them to stable and implement the missing `dispatchEvent` function along with `new Event()` and `new CustomEvent()`